support-frontend/test/selenium/subscriptions/ProductPagesSpec.scala

```diff
@@ -58,7 +58,10 @@ class ProductPagesSpec extends AnyFeatureSpec
 
   Feature("Digital Pack product page") {
     Scenario("Basic loading") {
-      val testUser = new PostDeployTestUser(driverConfig)
+      val testUser = IdapiTestUserRequest.getCookies match {
```

Comment:
I don't think this user needs cookies, we're just testing that the page loads and you don't need to be signed in for that.

Reply:
Nice catch - fixed!